src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20240416_add_sam_rac_tables.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="kfeldman" id="add_sam_rac_tables">
+        <createTable tableName="SAM_RESOURCE_ACCESS_CONSTRAINT">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="resource_id" type="BIGINT">
+                <constraints primaryKey="false" foreignKeyName="FK_SRAC_RESOURCE_ID" referencedTableName="SAM_RESOURCE" referencedColumnNames="id"/>
```

Comment:
I think this needs to be set to nullable?

Reply:
Based on how you set nullable=false on other columns it seems like you are going with the assumption that the default for columns is nullable. If that is the case then carry on. It might be good to specify it anyway though, would be annoying if you had to do a follow-up migration. I guess you could always run the migration locally and sanity check it.

Reply:
Yep agreed it's best to be clear and not rely on assumptions! I added an explicit nullable constraint on all the columns